docs/why_mf_next.md

```diff
@@ -74,9 +75,11 @@ But the stability requirements prevent any major cleanup.
   grammatical agreement requires words outside select / plural to change.
   See https://en.wikipedia.org/wiki/Agreement_(linguistics)
 - Patterns in the date / time / number placeholders are bad i18n, should use skeletons
-- No official support for gender. It can be done with `select`, but it
-  is not the same thing (same as the difference between an `enum` and integer/strings). Developers can use masculine/feminine, masc/fem, male/female, etc.
-- Formatting for “parameters” known at compile time
+- No official support for gender. It can be done with `select`, but that has problems for localization tooling:
+  strings can't account for the difference in gender categories across languages, and there are no standard “enum” identifiers:
+  developers use masculine/feminine, masc/fem, male/female, etc.
+- Formatting for “literals” known at compile time.
+  For example, literal numbers in a message need to be adjusted for the user's preferences for native vs ASCII digits and format.
```

Comment:
I don't understand what this is supposed to represent?

Reply:
This the case where you have Arabic *literal* text in "There are 98 departments" (note no input placeholders). There is no way in MF1.0 to change the 98 to Arabic numerals (from ASCII) according to user preferences.

Reply:
I didn't understand the original text, so tried to reformulate it into something that made sense to me.

Reply:
That was the line "Formatting for “parameters” known at compile time"

Reply:
If you have suggested for clarity ... Or we could just delete "Formatting for “parameters” known at compile time" and not replace by "Formatting for "literals"..."

Reply:
I think we should just remove this item.

Reply:
done